src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_fullyconnected_primitive.cpp

```diff
@@ -188,7 +186,7 @@ static DnnlPrimitiveAttrs createPrimitiveAttrs(const FCAttrs& attrs,
                                 isINT8,
                                 1 << 0,
                                 attrs.dequantizationScales,
-                                attrs.withBias,
+                                !memory.at(ARG_BIAS)->getDesc().empty(),
```

Comment:
Can be simplified if we decide to introduce empty() helper method to the Memory class as well.

Reply:
The thing is what is considered as the emptiness. Zero size, or special state. May be confusing. But may be resolved with more precise term.

Reply:
This is up to us to decide.
For me it correlates with the "empty()" methods used for containers.
You have container object, and there is nothing inside.
You have memory object, and there is nothing inside.